errors/index.js

```diff
@@ -0,0 +1,19 @@
+exports.handleNonexistentEndpoint = (req, res, next) => {
+    res.status(404).send({ msg: 'Not found' });
+}
+
+exports.handleCustomErrors = (err, req, res, next) => {
+    console.log(err);
+    next(err);
+}
+
+exports.handlePsqlErrors = (err, req, res, next) => {
+    console.log(err);
+    next(err);
+}
+
+
+exports.handleServerErrors = (err, req, res, next) => {
+    console.log(err);
+    res.status(500).send({ msg: 'Server error!' });
+}
```

Comment:
Good to have in general, just because some errors may slip past the other error handlers. This log can really come in handy for debugging